src/navigation/components/NavigationItem/NavigationItem.tsx

```diff
@@ -28,7 +28,7 @@ export interface NavigationItemProps {
 
 const Content: React.FC<{text: string; icon?: ImageProps}> = ({text, icon}) => (
     <Fragment>
-        {icon && <Image className={b('icon')} {...icon} />}
+        {icon && typeof icon !== 'string' && <Image className={b('icon')} {...icon} />}
```

Comment:
why do we need this fix?

Reply:
Because a new version of typescript requires a destructed item to be an Object. Icon type is complex: one of its type could be a string. String cannot be destructed